### PR TITLE
chore: Added fintech wallet fef

### PR DESCRIPTION
**Pull request by stackchain**
- [x] Add Fintech wallet to the .env vars
- [x] Refactor entries helpers 
- [x] Add tests to entries helpers 

Screenshot 2023-10-23 at 7 21 59 PM

### CODE REVIEW

File: apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx [hunk 1]
```diff
@@ -86,25 +87,31 @@ export const CreateOrder = () => {
...
+        const frontendFeeDepositAddress = isMainnet
+          ? Config['FRONTEND_FEE_ADDRESS_MAINNET']
+          : Config['FRONTEND_FEE_ADDRESS_PREPROD']
+        const frontendFeeEntry = makePossibleFrontendFeeEntry(frontendFee, frontendFeeDepositAddress)
+
+        const entries = frontendFeeEntry != null ? [orderEntry, frontendFeeEntry] : [orderEntry]
+
         createUnsignedTx({entries})
+      } else {
+        Alert.alert(strings.generalErrorTitle, strings.generalTxErrorMessage)
```
Comment (michalszorad) [inline apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx]: Maybe else block first ?

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx (hunk 1 above)
Comment (stackchain) [reply apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx]: Nice call, updating it.

Comment (stackchain) [reply apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx]: done, failing first.

**Review by banklesss — APPROVED** [review]

File: apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx [hunk 2]
```diff
@@ -88,7 +89,11 @@ export const CreateOrder = () => {
 
   const {createOrderData} = useSwapCreateOrder({
     onSuccess: (orderResponse: Swap.CreateOrderResponse) => {
-      if (orderResponse?.contractAddress === undefined || orderData.selectedPoolCalculation?.pool === undefined) {
+      if (
+        orderResponse?.contractAddress === undefined ||
+        orderData.selectedPoolCalculation?.pool === undefined ||
+        !isString(orderResponse?.datum)
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx:95]: the `datum` is not optional

**Pull request merged**